Source/WebCore/platform/graphics/texmap/TextureMapperLayer.cpp

```diff
@@ -63,6 +63,40 @@ struct TextureMapperLayer::ComputeTransformData {
     }
 };
 
+class TextureMapperFlattenedLayer : public TextureMapperPlatformLayer {
+public:
+    TextureMapperFlattenedLayer(const IntRect& rect)
+        : m_rect(rect)
+    { }
+
+    void bind(TextureMapper& textureMapper)
+    {
+        if (!m_texture)
+            m_texture = textureMapper.acquireTextureFromPool(m_rect.size(), { BitmapTexture::Flags::SupportsAlpha });
+        textureMapper.bindSurface(m_texture.get());
+    }
+
+    inline FloatRect layerRect() const
+    {
+        return FloatRect(m_rect);
```

Comment:
Why the explicit conversion? Can't you just expose the `IntRect` you stored?

Reply:
Fixed